### PR TITLE
Fixed all lint issues. Added coverage report output. Added test linting.

### LINKED ISSUE #291: !wolfram should use the env variable and UQCS app id
The app id for !wolfram is currently hard coded into the script and is also my personal app id. I don't really mind but I assume it would be preferable to have a specific UQCS one?

I believe the environment variable already exists as "WOLFRAM_APPID".

Happy to do this after exams.

### PR DESCRIPTION
**Pull request by mitchmcdee**
As per title :^)

Fixes #291

### CODE REVIEW

**Comment by mitchmcdee** [comment]
Note: this PR now adds strict line limits + import usage since I cleaned up all existing issues.

**Comment by TRManderson** [comment]
It seems to me like the verdict here is to prefer `# noqa` for unused imports from type comments - are we locking that in?

**Comment by mitchmcdee** [comment]
Means we can catch legitimate unused imports so yeah would prefer `# noqa` imo

File: test/conftest.py [hunk 1]
```diff
@@ -132,17 +132,23 @@ def mocked_channels_list(self, channel_type='channels', **kwargs):
         cursor = kwargs.get('cursor', 0)
         limit = kwargs.get('limit', 100)
 
+        def is_channel_type(channel, channel_type):
```
Comment (TRManderson) [inline test/conftest.py]: `is_channel_type = partial(dict.get, channel, default=False)`

Comment (mitchmcdee) [reply test/conftest.py]: ye nice

Comment (TRManderson) [reply test/conftest.py]: Nah doesn't work

Comment (TRManderson) [reply test/conftest.py]:
```python
>>> from functools import partial
>>> x = {}
>>> fn = partial(dict.get, x, default=False)
>>> fn(43)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: get() takes no keyword arguments
>>> fn(43)
```

Comment (mitchmcdee) [reply test/conftest.py]: kek. good to merge then?

**Pull request merged**